.github/workflows/ci.yml

```diff
@@ -135,6 +135,10 @@ jobs:
       - name: checkout
         uses: actions/checkout@v4
 
+      - name: install typescript
```

Comment:
I'm not sure this is necessary due to this fix (merged) https://github.com/tari-project/tari-dan/pull/1260/files#diff-04646d5069f9923612c7ce0f122758738e2f809a5ff3628e25fc691e683dca93

Reply:
ah okay, yeah yesterday the branch was broken, let me rollback